Resources/templates/responsive/channel/call/layout.php

```diff
@@ -23,6 +23,13 @@
 <?php
 $this->replace();
 
+$this->section('favicon');
+
+if ($this->channel->label): ?>
+        <link rel="icon" type="image/x-icon" href=<?= $this->channel->label->getLink(16,16,true) ?>
```

Comment:
Missing closing `>` for the `link` tag. Maybe also quote the href value for better readability.

```php
<link ... href="<?= $this->channel->label->getLink(16, 16, true) ?>">
```